### PR TITLE
Show exposures <1s as 1/N instead of 0.X''

### PR DESCRIPTION
**Pull request by pedrocr**
It seems much nicer to see exposures as 1/N instead of 0.X''. Technically this will "regress" some history stacks as watermark can use these values but it seems like a minor issue.

### CODE REVIEW

File: src/common/image.c [hunk 1]
```diff
@@ -272,7 +272,7 @@ void dt_image_path_append_version(int imgid, char *pathname, size_t pathname_len
 
 void dt_image_print_exif(const dt_image_t *img, char *line, size_t line_len)
 {
-  if(img->exif_exposure >= 0.1f)
+  if(img->exif_exposure >= 1.0f)
     snprintf(line, line_len, "%.1f'' f/%.1f %dmm iso %d", img->exif_exposure, img->exif_aperture,
```
Comment (LebedevRI) [inline src/common/image.c]: Probably s/''/"/
(double ' vs. single ")

Comment (pedrocr) [inline src/common/image.c]: The notation for seconds is '' (two), just one means minutes:

http://english.stackexchange.com/questions/114205/english-notation-for-hour-minutes-and-seconds

I didn't touch that but I guess that was the rationale behind it.

Comment (LebedevRI) [inline src/common/image.c]: Or:
https://en.wikipedia.org/wiki/Prime_(symbol)
The double prime (″) represents inches (in), arcseconds (as), and seconds (s). However, for convenience, a (") (double quotation mark) is commonly used.

Comment (pedrocr) [inline src/common/image.c]: Sounds fine to me

**Comment by LebedevRI** [comment]
Why it should *not* be switched at 1s:

actual in-camera setting | displayed as
------------ | -------------
1/4 | 1/4
0"3 | 1/3
0"4 | 1/2
0"5 | 1/2
0"6 | 1/2
0"8 | 1/1
1" | 1"

**Comment by pedrocr** [comment]
I had a look at how the cameras I had around do it and implemented the following cases:
 * As long as it is as fast or faster that 0.3s use 1/N
 * From 0.4s up to 3.9s use N"M
 * From 4s up use just N"

This looks good to me but the 0.3s can be done either way really (0"3 and 1/3 are equally as good I'd say).

**Comment by houz** [comment]
At least make the strings translatable. In the German translations I am always using correct typography, so I will definitely use the second sign here.

**Comment by pedrocr** [comment]
@houz what do you want to make translatable? Besides the values we only have "/mm/iso

**Comment by houz** [comment]
The whole strings containing the `\"` which you `snprintf()`.

**Comment by pedrocr** [comment]
@houz but why would we translate those? There's nothing there that's language specific. Let's just find something we like and stay with it.

**Comment by houz** [comment]
As I said, the German translation always used correct typography, so i would use the second sign and not the `"`. I also change all quotes.

**Comment by pedrocr** [comment]
Ok, this now uses the double prime symbol and I've tested it's correct in 0.1s increments from 0.1s to 4.5s which covers all the cases. As far as I know this is ready to merge.

**Comment by pedrocr** [comment]
I've tested this from 0.01s to 4.50s in 0.01s increments now and fixed an error where we could end up displaying 0"10 instead of 1"0.

**Pull request merged**